content/apps/multiple-instances.md

```diff
@@ -13,6 +13,8 @@ There are three ways to start multiple app instances in Cloud Foundry:
 2.  In the `manifest.yml`
 3.  Using the `cf scale` command
 
+Note that scaling to multiple instances in cloud.gov should be scheduled in more than one AWS availability zone.
```

Comment:
I'd like to avoid allowing specific IaaS provider references to proliferate in the docs, and I think we can be clearer: "When you scale your app by running multiple instances, cloud.gov attempts to schedule those instances across availability zones to maximize your app's resiliency."